decidim-proposals/app/controllers/decidim/proposals/admin/proposal_answers_controller.rb

```diff
@@ -44,6 +63,42 @@ def skip_manage_component_permission
         def proposal
           @proposal ||= Proposal.where(component: current_component).find(params[:id])
         end
+
+        def proposals
+          @proposals ||= Proposal.where(component: current_component).where(id: params[:proposal_ids])
+        end
+
+        def template
+          @template ||= Decidim::Templates::Template.find(params[:template][:template_id])
+        end
+
+        def bulk_answer_form(proposal)
+          @bulk_answer_form ||= ProposalAnswerForm.from_params(prepare_answer_form_params(template, proposal, current_user)).with_context(current_component:, current_organization:)
```

Comment:
```suggestion
          @bulk_answer_form ||= form(ProposalAnswerForm).from_params(prepare_answer_form_params(template, proposal, current_user))
```
This will add the context automaically